src/components/chart/_macro.njk

```diff
@@ -87,11 +87,10 @@
             {# If the chart is displayed inside an iframe, then the fallback image is not displayed here, as it will be provided in the iframe content instead #}
             {% if params.fallbackImageUrl and not params.iframeUrl %}
                 <noscript data-fallback-image>
-                    {# alt text is empty as the description is provided above #}
+                    {# Note that a more detailed description of the chart is provided in the audio description above #}
                     <img
                         src="{{ params.fallbackImageUrl }}"
-                        aria-describedby="{{ audioDescriptionId }}"
-                        alt=""
+                        alt="Fallback image for the chart as JavaScript is disabled"
```

Comment:
This text should probably be configurable and have a default instead of being hardcoded. It would mean that other languages will be able to be used for example

Reply:
That's now configurable with a default value if not set.